pkg/webhook/stats_reporter.go

```diff
@@ -57,6 +59,25 @@ func (r *reporter) ReportValidationRequest(ctx context.Context, response request
 	return r.reportRequest(ctx, response, admissionStatusKey, validationResponseTimeInSecM.M(d.Seconds()))
 }
 
+func (r *reporter) ReportValidationRequestAndType(ctx context.Context, response requestResponse, isDryRun *bool, d time.Duration) error {
+	var dryRun string
+	if *isDryRun {
+		dryRun = "dryrun"
+	} else {
+		dryRun = "finalrun"
```

Comment:
for simplicity's sake, we should keep the original enforcement action name (or unrecognized).

This function can help:

https://github.com/open-policy-agent/gatekeeper/blob/65f82459aaa536c3bc2a2829bd138351caefa3e8/pkg/util/enforcement_action.go#L43-L59

Reply:
We won't have any indication about `deny` request, right? `admissionRequest` only has `DryRun` property to indicate if it's dryRun or not. So the classification would change as if not `dryRun` then unrecognized? is this assumption correct? or did you mean something else? @maxsmythe

Reply:
@JaydipGabani this is for constraint's enforcement action, not admission request dryrun. https://open-policy-agent.github.io/gatekeeper/website/docs/violations/#dry-run-enforcement-action

Reply:
Sorry, I had misread the original bug I thought we were talking about "dryrun" the enforcement action.

Reply:
nvm, wordsmithing is moot now that the PR has changed to a bool label